lib/documents/schemas/residential_property_tribunal_decisions.json

```diff
@@ -68,179 +68,179 @@
         },
         {
-          "label": "Housing Act 2004 & Housing and Planning Act 2016 - Management Orders",
-          "value": "housing-act-2004-housing-and-planning-act-2016---management-orders"
+          "label": "Housing Act 2004 and Housing and Planning Act 2016 - Management orders",
+          "value": "housing-act-2004-and-housing-and-planning-act-2016---management-orders"
         },
         {
-          "label": "Housing Act 2004 & Housing and Planning Act 2016 - Overcrowding Notices",
-          "value": "housing-act-2004-housing-and-planning-act-2016---overcrowding-notices"
+          "label": "Housing Act 2004 and Housing and Planning Act 2016 - Overcrowding notices",
+          "value": "housing-act-2004-and-housing-and-planning-act-2016---overcrowding-notices"
         },
         {
-          "label": "Leasehold Disputes (Management) - Service Charges",
+          "label": "Leasehold disputes (management) - Service charges",
           "value": "leasehold-disputes-management---service-charges"
         },
         {
-          "label": "Leasehold Disputes (Management) - Administration Charges",
+          "label": "Leasehold disputes (management) - Administration charges",
           "value": "leasehold-disputes-management---administration-charges"
         },
         {
-          "label": "Leasehold Disputes (Management) - Appointment of Manager",
+          "label": "Leasehold disputes (management) - Appointment of manager",
           "value": "leasehold-disputes-management---appointment-of-manager"
         },
         {
-          "label": "Leasehold Disputes (Management) - Right to Manage",
+          "label": "Leasehold disputes (management) - Right to Manage",
           "value": "leasehold-disputes-management---right-to-manage"
         },
         {
-          "label": "Leasehold Disputes (Management) - Variation of Leases",
+          "label": "Leasehold disputes (management) - Variation of leases",
           "value": "leasehold-disputes-management---variation-of-leases"
         },
         {
-          "label": "Leasehold Disputes (Management) - Breach of Lease/Forfeiture",
+          "label": "Leasehold disputes (management) - Breach of lease/forfeiture",
           "value": "leasehold-disputes-management---breach-of-lease-forfeiture"
         },
         {
-          "label": "Leasehold Disputes (Management) - Estate Charges/Estate Management Schemes",
+          "label": "Leasehold disputes (management) - Estate charges/estate management schemes",
           "value": "leasehold-disputes-management---estate-charges-estate-management-schemes"
         },
         {
-          "label": "Leasehold Enfranchisement & Extension - Flats - Lease Extension ",
-          "value": "leasehold-enfranchisement-extension---flats-lease-extension-"
+          "label": "Leasehold enfranchisement and extension - Flats - lease extension",
+          "value": "leasehold-enfranchisement-and-extension---flats-lease-extension-"
```

Comment:
Extra space at the end here (not your fault, it was in the original, but we should tidy it now).